src/bankAccount.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+BankAccount.prototype = {
+  getInfo: function() {
+    return `Name: ${this.name}, Amount: ${this.amount}$`;
+  },
+  addMoney: function(value, assignment) {
+    this.amount += value;
+    this.accountHistory.push(`${assignment}: ${value}`);
+  },
+  withdrawMoney: function(value, purpose) {
+    this.amount -= value;
+    this.accountHistory.push(`${purpose}: -${value}`);
+  },
```

Comment:
функции одинаковые, одна добавляет, вторая отнимает, старайся называть переменные схожим образом
```suggestion
  addMoney: function(value, operation) {
    this.amount += value;
    this.accountHistory.push(`${assignment}: ${value}`);
  },
  withdrawMoney: function(value, operation) {
    this.amount -= value;
    this.accountHistory.push(`${purpose}: -${value}`);
  },
```